app/gateway-oss/2.5.x/getting-started/adding-consumers.md

```diff
@@ -30,7 +30,7 @@ $ curl -i -X POST \
   --data "username=Jason"
 ```
 
-You should see a response similar to the one below:
+You can see a response similar to the one below:
```

Comment:
```suggestion
You should see a response similar to the one below:
```

Changing it back to "should" because that implies that the action was potentially successful.